charts/ratify/templates/provider.yaml

```diff
@@ -11,3 +11,19 @@ spec:
   {{- if .Values.provider.auth }}
   caBundle: {{ required "You must provide .Values.provider.tls.cabundle when .Values.provider.auth is set" .Values.provider.tls.cabundle }}
   {{- end }}
+
+---
+
+apiVersion: externaldata.gatekeeper.sh/v1alpha1
```

Comment:
This is beta in `v3.11` release of gatekeeper

Reply:
added version based on selected gk version